app/assets/javascripts/lookup_keys.js

```diff
@@ -324,9 +324,43 @@ function fill_in_matchers() {
         .attr('selected', 'selected');
       matcher_value.val(key_value[1]);
     }
+
+    $(matcher_value).data('initialValue', matcher_value.val());
+    $(matcher_key).data('initialValue', matcher_key.find(":selected").text());
+
+  });
+
+  $('.input-group textarea[data-property="value"]').each(function(idx, element) {
+    $(element).data('initialValue', $(element).val());
   });
 }
 
+function matcherKeyChanged(element) {
```

Comment:
could we add these new functions to webpacked js on `window.tfm` object in some subkey?
We will probably need new file, but I think we should not use the global functions anymore.
Something like `webpack/assets/javascripts/foreman_lookup_keys.js` would do nicely IMHO.